src/main/java/org/embl/mobie/lib/create/RemoteMetadataCreator.java

```diff
@@ -112,11 +108,11 @@ private void addRemoteMetadataForImage( String datasetName, String imageName ) t
                         datasetName,
```

Comment:
Above (line 95), only `relativePath` are allowed when projects are uploaded to S3 (to prevent local links to images outside of the MoBIE project from being broken on upload). I guess an absolute path that was to S3 would work though - do you want to allow this case?

Reply:
Yes, I think we should allow the absolute path linking to S3.